api/jobs.py

```diff
@@ -231,6 +251,20 @@ def next(self):
         if result == None:
             self.abort(400, 'No jobs to process')
 
+        # Second, update document to store formula request.
+        result = self.app.db.jobs.find_one_and_update(
+            {
+                '_id': result['_id']
+            },
+            { '$set': {
+                'request': generate_formula(result['intent'])}
+            },
+            return_document=pymongo.collection.ReturnDocument.AFTER
+        )
+
+        if result == None:
```

Comment:
While this yields the correct result, the common pythonism for equality to ```None``` is ```result is None```. Also in line 251 above.

I often use the more casual ```if not result```, although that evaluates to ```true``` in many more cases, such as ```{}, [], '', 0```.